<launch>



  <!--  <node pkg="hal_control" type="drive_control.py" name="drive_control">
       <remap from="rover_state_cmd" to="rover_state_cmd" />
    </node>-->
 <!-- <node respawn="true" pkg="joy"
        type="joy_node" name="joy_drive" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <remap from="joy" to="joy_drive"/>
  </node>-->

  <!--
  <node pkg="base" type="base" name="rover_hub">
    <param name="num_cam" value="2"/>
  </node>
  -->

  <node name="main_camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/rover/cameras/Logitech_C922x" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="yuyv" />
    <param name="framerate" value="25" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="sensor" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/rover/cameras/forearm" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="framerate" value="15" />
    <param name="io_method" value="mmap"/>
    </node>

<!--  <node name="usb_cam0_process" pkg="image_proc" type="image_proc" output="screen">
    <remap from="/image_raw" to="/usb_cam0/image_raw"/>
  </node> -->

</launch>
